### DISCUSSION TITLE
New release

### DISCUSSION THREAD

**Discussion by tjwallace** [backube/volsync#1384] [General]
Are there any plans to cut a new release soon? I would like to use the `moverAffinity` feature merged in https://github.com/backube/volsync/pull/1340. Thanks!

**Comment by tesshuflower** [comment]
Sorry we're a little slow - I depend on the schedule of a parent product so atm it's looking like end of October for volsync v0.11.0

**Reply by onedr0p** [reply]
@tesshuflower sorry this is due to my ignorance on your processes but I am curious, couldn't the parent product pin to previous releases of volsync, e.g. say it can stay on v0.10.0 so it doesn't hold up the release process here? If not, maybe there could be some way of creating a pre-release that people could start testing and using without interfering with the parent product? 

Not to rush you but there are a lot of useful features coming in v0.11.0 that a bunch of us would love to start testing and using.

**Reply by tesshuflower** [reply]
What we actually have is a separately built downstream version of VolSync, and I'd rather not re-version those. This is in testing now but does also depend release wise with  another parent product.

But, I agree it's not great that you're getting held up at this point. Right now if you use the `release-0.11` branch, this will be what ends up in v0.11.0 (barring some last minute fix).  Which also means the latest images tagged `release-0.11` [here](https://quay.io/repository/backube/volsync?tab=tags) are the latest that should be close to what you get with v0.11.0:  

Let me get back to you though, maybe we could start doing pre-release tags to make it easier to consume these builds.

**Reply by onedr0p** [reply]
It wouldn't be as simple as swapping the image as I am installing volsync with the helm chart so unless the chart is also pre-released my cluster wouldn't get the new CRDs.

**Comment by onedr0p** [comment]
@tesshuflower the below should satisfy my needs for now and it's easy enough to switch back and forth between referencing a Git repo or Helm repo.

For anyone coming here looking to try out unreleased release branches, I am doing this with Flux:

https://github.com/onedr0p/home-ops/pull/8255

**Comment by tesshuflower** [comment]
@onedr0p I think we can consider creating rc releases if that seems useful.  Most likely something like `vX.Y.Z-rc.1`.

I have to check that helm supports semantic versioning ok in this way, but I think it should work - it sounds like if we published a `v.X.Y.Z-rc.1` helm chart version, you might need to use the `--devel` flag in order to deploy these sorts of releases.

**Reply by onedr0p** [reply]
That would be amazing! flux should be able to deploy it either way from my end 😄

**Reply by tesshuflower** [reply]
ok we've published a pre-release called `v0.11.0-rc.1`.  With any luck the actual release v0.11.0 should come out end of next week sometime.

There is a `v0.11.01-rc.1` helm chart version available now as well at the usual [location](https://backube.github.io/helm-charts/).

Note to install/upgrade to it using helm you may need to set the version flag explicitly (`--version 0.11.0-rc.1`) or pass in the `--devel` flag.

**Reply by onedr0p** [reply]
This is really amazing @tesshuflower thanks for all the work on getting this out so quick ❤️

**Reply by tesshuflower** [reply]
It was a good suggestion to make pre-releases - glad you're willing to try them out! :rocket:

**Reply by onedr0p** [reply]
I've confirmed the new features you added for restic are working great, these are some great QoL improvements
